### PR TITLE
Add Boom 3D v1.1

### PR DESCRIPTION
**Pull request by 0rax**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
This looks like it's a newer version of [boom](https://github.com/caskroom/homebrew-cask/blob/master/Casks/boom.rb)

boom3d

**Comment by 0rax** [comment]
@commitay Wasn't sure what to do exactly as its 2 different softwares in the end. Both have the same main feature but are still separate app.

You can still use and buy Boom2 (`boom.rb`) and I really do think that some / most of the current users that do not wants the new features will upgrade their licenses.

**Comment by commitay** [comment]
We can remove `boom` for now and it can be added to `caskroom/versions` later if anyone wants it.

https://github.com/caskroom/homebrew-cask/pull/38794
___

`boom3d` will probably need a `quit`/`signal` and `kext` uninstall added similar to `boom`.

You can check this with the dev scripts in `$(brew --repository)/Library/Taps/caskroom/homebrew-cask/developer/bin`

```
./list_loaded_kext_ids
./list_running_app_ids
```

**Comment by 0rax** [comment]
@commitay I am gonna check this, though during my tests it seemed that the `launchtl` stopped everything cleanly on `brew cask uninstall`. By that I mean the helper + tray icon are gone and the sound switches back to what it is before the installation.

**Comment by 0rax** [comment]
Just updated to add the quit option. This make sure that the sound is restore to normal before the app is uninstalled (otherwise it can make your mac go mute).

Just verified, everything is cleaned properly in term of launchtl and kexts after a uninstall and the `BoomAudio` sound device is properly removed after a reboot.

**Comment by commitay** [comment]
Thanks @0rax!

**Review by commitay — APPROVED** [review]

File: Casks/boom3d.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask 'boom3d' do
```
Comment (reitermarkus) [inline Casks/boom3d.rb]: Should be `boom-3d`.

**Review by reitermarkus — DISMISSED** [review]

**Pull request merged**